app/assets/javascripts/application.js

```diff
@@ -37,3 +28,18 @@ function stripeResponseHandler(status, response) {
     $form.get(0).submit();
   }
 };
+
+$(document).ready(function() {
+  $("#use_saved_card").on("click", toggleCreditCardFields);
+});
+
+function toggleCreditCardFields(){
```

Comment:
I did that elsewhere, but I apparently missed that here. Thanks for pointing that out.